src/web/mjs/connectors/templates/CoreView.mjs

```diff
@@ -9,9 +9,9 @@ export default class CoreView extends Connector {
         super.label = undefined;
         this.tags = [];
         this.url = undefined;
-        this.requestOptions.headers.set('x-user-agent', 'Mozilla/5.0 (iPod; CPU iPhone OS 14_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/87.0.4280.163 Mobile/15E148 Safari/604.1');
+        this.requestOptions.headers.set('x-user-agent', 'Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36');
```

Comment:
What happens when completely remove forced user agent and use the automatically assigned user agent instead?

Reply:
The problem cannot be reproduced now because it loads chapters normally with previous user agent over time.
Next manga's update date is 3/10, so I'll check it again then.